### PR TITLE
add two missing constants in py2.7

### PR DESCRIPTION
**Pull request by schriftgestalt**
The system pyobjc bridge misses this two

### CODE REVIEW

**Comment by justvanrossum** [comment]
This is confusing: the import is not used, and is only used for its side effect. Also, I don't like poking values into an existing module. Better to do something like this in `vanillaBase.py`:

```python
try:
    from AppKit import NSLayoutAttributeLastBaseline, NSLayoutAttributeFirstBaseline
except ImportError:
    NSLayoutAttributeLastBaseline = 11
    NSLayoutAttributeFirstBaseline = 12
```

**Comment by schriftgestalt** [comment]
I tried to keep the mess out of the main source file. And the py23 is a kind of a hack anyway. And it will be removed soon.

**Comment by justvanrossum** [comment]
I'm not going to argue, but like this I won't accept the PR.

File: Lib/vanilla/py23.py [hunk 1]
```diff
@@ -30,3 +30,10 @@
 except ImportError:
     def python_method(arg):
         return arg
+try:
+    print "import NSLayoutAttributeLastBaseline"
```
Comment (benkiel) [inline Lib/vanilla/py23.py]: Shouldn't be here, adds noise

File: Lib/vanilla/vanillaBase.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import platform
 from distutils.version import StrictVersion
+import py23
```
Comment (benkiel) [inline Lib/vanilla/vanillaBase.py]: shouldn't this be `import vanilla.py23`? See other cases of this import

**Comment by benkiel** [comment]
Agree with @justvanrossum, this is better in `vanillaBase.py` than in `py23`. Even if this is the PR, see notes on lines.

**Comment by justvanrossum** [comment]
Thanks @schriftgestalt!

**Comment by schriftgestalt** [comment]
I forgot to import the second constant. Sorry about that. I’m still recovering from my 50 hour trip back from ATypI.

**Comment by justvanrossum** [comment]
Ah, you're right, I missed that, too. I'll fix it.

**Pull request merged**